packages/teams-js/src/public/tasks.ts

```diff
@@ -42,10 +42,10 @@ export namespace tasks {
       : undefined;
     if (taskInfo.card === undefined && taskInfo.url === undefined) {
       ensureInitialized(runtime, FrameContexts.content, FrameContexts.sidePanel, FrameContexts.meetingStage);
-      sendMessageToParent('tasks.startTask', [taskInfo as DialogInfo], submitHandler);
+      sendMessageToParentWithVersion('v1', 'tasks.startTask', [taskInfo as DialogInfo], submitHandler);
```

Comment:
You're using v1 both here and in dialogs. Should dialogs be v2 so we can tell those calls apart from these ones?

Reply:
I have made an update and now v1 is used here for tasks.startTask - with `taskInfo` param and an optional `submitHandler` - and for dialog.url.open (its replacement), it should be 'v2' now.